features/checkout/addressing_order/choosing_province_for_country.feature

```diff
@@ -6,7 +6,7 @@ Feature: Choosing province for country
 
     Background:
         Given the store operates on a single channel in "United States"
-        And country "United States" has the "New York" province with "NY" code
+        And the country "United States" has the "New York" province with "NY" code
```

Comment:
1) And the "United States" country has the "New York" province with "NY" code
2) It is not totally true, I guess. As far as I remember, countries don't have `the` before theirs name. The exception is when the name of a country suggest larger number of smaller pieces that together are country. /cc @TheMadeleine